content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,139 @@
+---
+title: OpenTelemetry Collector Configuration Best Practices
+linkTitle: Collector config
+description:
+  Follow best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When setting up configuration for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your OTel Collector instance.
+
+<!--
+TODO: SHOULD only enable the minimum required components. What are those?
+--->
+
+## Receivers and Exporters
+
+- Receivers and Exporters can be either push or pull-based. In either case, the
+  connection established SHOULD be over a secure and authenticated channel.
+- Unused receivers and exporters SHOULD be disabled to minimize the attack
+  vector of the Collector.
+- Receivers and Exporters MAY expose buffer, queue, payload, and/or worker
+  settings via configuration parameters. If these settings are available, users
+  should proceed with caution before modifying the default configuration values.
+  Improperly setting these values may expose the OpenTelemetry Collector to
+  additional attack vectors including resource exhaustion.
+
+It is possible that a receiver MAY require the OpenTelemetry Collector run in a
+privileged mode in order to operate, which could be a security concern, but
+today this is not the case.
+
+### Safeguards against denial of service attacks
+
+Users should bind receivers' servers to addresses that limit connections to
+authorized users, so that their Collectors aren't exposed to the public internet or to wider networks than necessary.
```

Comment:
```suggestion
You should bind receivers' servers to addresses that limit connections to
authorized users, so that their Collectors aren't exposed to the public internet or to wider networks than necessary.
```